hail/src/main/scala/is/hail/expr/ir/BlockMatrixIR.scala

```diff
@@ -217,6 +218,7 @@ case class BlockMatrixMap(child: BlockMatrixIR, eltName: String, f: IR) extends
   }
 }
 
+//FIXME: fix sparsity after refactor lands
```

Comment:
does this still need to be fixed?

Reply:
yeah---this one and the other Map node still need #8072 to land before I can handle it properly (because of all the introspection that happens within the node's execution to determine the sparsity at runtime). I can wait for that to land and merge my other branch's work into here afterwards, if you'd prefer, but it also won't affect anything that's currently running.